### ISSUE TITLE
OOM for training

### ISSUE THREAD

**Issue by wanmeiziyoulan** [Megvii-BaseDetection/YOLOX#805]
Hi, I try to train on my dataset, the running command is _python tools/train.py -f exps/default/yolov3.py -d 4 -b 32 --fp16 -c YOLOX_outputs/yolov3_17/latest_ckpt.pth_. The first 17 epochs are relatively normal. When running to the 18th epochs, loss begins to grow and report the errors in the following pictures.
![image](https://user-images.githubusercontent.com/18022941/137836747-85834687-414c-4a7e-b86f-a51ea5d43e8c.png)

I learned from issues that the latest code has fixed this problem. What is the reason for its recurrence？
Looking forward to your reply soon!  Thanks a lot!

**Comment by leetesua** [comment]
same here. No clue why.

**Comment by mxksowie** [comment]
I'm facing a similar issue. Currently, I've found that reducing the batch size has prevented the conf loss from blowing up. I'm still not sure why this works.

**Comment by leetesua** [comment]
I comment out the learning rate policy section in config file and this error disappeared.

**Comment by mxksowie** [comment]
Does this mean you are using an absolute learning rate now?
I've tried both increasing and decreasing `base_lr_per_img` but it did not work for me.
However, that seems to have worked in this other issue: #830

**Comment by leetesua** [comment]
> Does this mean you are using an absolute learning rate now? I've tried both increasing and decreasing `base_lr_per_img` but it did not work for me. However, that seems to have worked in this other issue: #830
yes i am using a fix number (0.01) as lr rate.  I also changed to yolox-l, (I was using yolox-x, the biggest version)
